fe1-web/components/eventList/events/Event.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import { View, ViewStyle, TouchableOpacity } from 'react-native';
+import PropTypes from 'prop-types';
+
+import { Spacing } from 'styles/index';
+import PROPS_TYPE from 'res/Props';
+
+import ParagraphBlock from 'components/ParagraphBlock';
+import styleEventView from 'styles/stylesheets/eventView';
+import TextBlock from 'components/TextBlock';
+import EventMeeting from './EventMeeting';
+import EventRollCall from './EventRollCall';
+import ListCollapsibleIcon from '../ListCollapsibleIcon';
+// import RollCallEventOrganizer from './RollCallEventOrganizer';
+
+/**
+ * The Event item component: display the correct representation of the event according to its type,
+ * otherwise display its name and in all cases its nested events
+*/
+const Event = (props: IPropTypes) => {
+  const { event } = props;
+  const { renderItemFn } = props;
+  const isOrganizer = false; // FIXME ?
+
+  const [toggleChildrenVisible, setToggleChildrenVisible] = useState(false);
+
+  const hasChildren = () => event.children && event.children.length !== 0;
+
+  const toggleChildren = () => {
+    if (hasChildren()) setToggleChildrenVisible(!toggleChildrenVisible);
```

Comment:
isn't this just:
```suggestion
    setToggleChildrenVisible(hasChildren() && !toggleChildrenVisible);
```
 ?

Reply:
@pierluca I remember testing this yesterday without success. Tried it again just to make sure. Turns out it works if I write it like this
```javascript
setToggleChildrenVisible(!!hasChildren() && !toggleChildrenVisible); // double inversion
```

This is strange since `hasChildren` (highlighted above) has to return a boolean even with the javascript strange boolean operation standards we talked about yesterday 🤔